### DISCUSSION TITLE
Best 3/8" tubing for flowcell

### DISCUSSION THREAD

**Discussion by Telemachus1977** [tagyoureit/nodejs-poolController#1052] [Q&A]
I've learned that the Everbilt  3/8 in polyethylene tubing that you get at Home Depot lasts two summers maximum before it becomes brittle and breaks apart, ensuring that your pump is running dry and your flow cell is not functioning.

I am thinking of replacing it with blue John Guest LLDPE.

The stuff that comes with commercial flow cells appears to be black, not sure what material it is. 

Suggestions on best material for hot socal sun?

**Comment by tagyoureit** [comment]
Black may last a bit longer.  I believe silicone would be your best bet.

**Comment by TopJim** [comment]
I use John Guest brand black LLDPE on my flow cells and acid system, 1/4” though. Going strong after a year so far.
